tests/google_sheets/test_google_sheets_pipeline.py

```diff
@@ -1,36 +1,33 @@
 import pytest
 import dlt
-from pipelines.google_sheets.google_sheets import google_spreadsheet
+from pipelines.google_sheets import google_spreadsheet
 from tests.utils import ALL_DESTINATIONS, assert_load_info
 
 
-TEST_SPREADSHEETS = ["https://docs.google.com/spreadsheets/d/1HhWHjqouQnnCIZAFa2rL6vT91YRN8aIhts22SUUR580"]
+#TEST_SPREADSHEETS = ["...."] - this value is now saved in config.toml
 # list expected tables and the number of columns they are supposed to have
 ALL_TABLES = ["all_types", "empty_row", "empty_rows", "has_empty", "hole_middle", "inconsistent_types", "more_data", "more_headers_than_data", "NamedRange1", "only_data", "only_headers",
-              "sheet1", "sheet2", "sheet3", "sheet4", "two_tables"]
+              "Sheet 1", "sheet2", "sheet3", "sheet4", "two_tables"]
 COL_NUMS = [6, 5, 5, 5, 7, 5, 4, 8, 4, 2, 2, 4, 5, 9, 1, 9]
 ALL_TABLES_LOADED = ["all_types", "empty_row", "empty_rows", "has_empty", "hole_middle", "inconsistent_types", "more_data", "more_headers_than_data", "named_range1", "only_data", "only_headers",
-                     "sheet1", "sheet2", "sheet3", "sheet4", "spreadsheet_info", "two_tables"]
+                     "sheet_1", "sheet2", "sheet3", "sheet4", "spreadsheet_info", "two_tables"]
 
 
 def create_pipeline(destination_name, dataset_name, full_refresh=True, range_names=None, get_sheets=True, get_named_ranges=True):
 
     pipeline = dlt.pipeline(destination=destination_name, full_refresh=full_refresh, dataset_name=dataset_name)
-    data = google_spreadsheet(spreadsheet_identifier=TEST_SPREADSHEETS[0], range_names=range_names, get_sheets=get_sheets, get_named_ranges=get_named_ranges)
+    data = google_spreadsheet(range_names=range_names, get_sheets=get_sheets, get_named_ranges=get_named_ranges)
     info = pipeline.run(data)
     assert_load_info(info)
```

Comment:
I'm actually not really sure about this, because assert_load_info is a simple check that all expected loads were loaded in the pipeline. There also isn't way to do this without return info as well and ending up with a lot of repeated code for all testers.